microsoft-edge/extensions-chromium/index.md

```diff
@@ -22,6 +22,8 @@ An extension should include at least the following features:
 
 To work directly with a specific part of the browser, such as a window or tab, you must send API requests, and must often reference the browser by name.
 
+<!-- todo: equivalent of landing page cards -->
```

Comment:
todo: equivalent of landing page cards, within Intro section of Intro article.
need to discuss the design objectives, to give alternate options for implementation format.
@sivMSFT 

rendered article intro:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/?branch=pr-en-us-3265

Reply:
closing this comment.  
pls look at new Landing page for extensions above the Overview article. 
consider if architecture diagram needed or flow diagram needed, where you had envisioned cards listing articles.  if so, should that be a separate PR?
@sivMSFT @bhuvanapriyap

if needed, enter fresh PR Comment.  closing the present comment so that the PR is ready to merge/publish & assumed scope of PR is just to add new landing page, not to revise content or doc design of existing Intro article.

rendered article intro:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/?branch=pr-en-us-326